fifteen_min/src/viewer.rs

```diff
@@ -283,44 +286,32 @@ pub fn draw_star(ctx: &mut EventCtx, b: &Building) -> GeomBatch {
 }
 
 fn build_panel(ctx: &mut EventCtx, app: &App, start: &Building, isochrone: &Isochrone) -> Panel {
-    let mut rows = vec![Line("15-minute neighborhood explorer")
-        .small_heading()
-        .into_widget(ctx)];
-
-    rows.push(map_gui::tools::change_map_btn(ctx, app));
-
-    rows.push(
+    let mut rows = vec![
```

Comment:
The title screen can generally be accessed by clicking the logo:
![Screenshot from 2021-11-04 14-18-16](https://user-images.githubusercontent.com/1664407/140421556-c16dd311-1fd9-43d0-9a33-71bfdcab72e7.png)
An exception is using the "home" button in sandbox mode:
![Screenshot from 2021-11-04 14-19-04](https://user-images.githubusercontent.com/1664407/140421616-5c248973-f5d2-4f14-bd6d-3990fb4a308c.png)
(That still goes to the title screen, just uses a different icon)